nntrainer/layers/input_layer.cpp

```diff
@@ -22,15 +22,40 @@
  */
 
 #include <input_layer.h>
+#include <layer_internal.h>
 #include <nntrainer_error.h>
 #include <nntrainer_log.h>
 #include <parse_util.h>
 
 namespace nntrainer {
 
-void InputLayer::setProperty(const PropertyType type,
+void InputLayer::setProperty(const std::vector<std::string> &values) {
+  /// @todo: deprecate this in favor of loadProperties
```

Comment:
Maybe add LayerImpl::setProperty(values) to up or down?

Reply:
up or down?

Reply:
Top of this function or bottom of this function :)

Reply:
Resolved offline, that there is no problem :)